i18n/ko.json

```diff
@@ -320,60 +320,60 @@
     "continueAnyway": "무시하고 계속",
     "directory": "디렉터리",
     "doesNotExist": "존재하지 않습니다.",
-    "HubAlreadyExists": "같은 이름의 Hub가 있음",
+    "hubAlreadyExists": "같은 이름의 Hub가 있음",
     "noSuchFileFound": "해당 파일을 찾을 수 없음:",
-    "pleaseChangeName": "위 Hub 이름을 변경하십시오.",
-    "selectDefaultPlayer": "선호하는 동영상 플레이어를 선택하세요.",
-    "selectPreviousHub": "이전에 저장한 동영상 Hub 파일을 선택하세요.",
+    "pleaseChangeName": "Hub 이름을 변경하세요.",
+    "selectDefaultPlayer": "기본으로 사용할 동영상 플레이어를 선택하세요.",
+    "selectPreviousHub": "이전에 저장된 동영상 Hub 파일을 선택하세요.",
     "selectRootFolder": "원본 동영상 폴더 선택",
     "videoFolderNotFound": "원본 동영상 폴더를 찾을 수 없음:"
   },
   "TAGS": {
-    "add": "Add",
+    "add": "추가",
     "addSuccess": "추가됨",
-    "addTag": "Add tag",
+    "addTag": "태그 추가",
     "alreadyExists": "태그가 이미 있음",
-    "any": "any",
-    "closeTagTray": "태그 트레이 닫기",
-    "delete": "delete mode",
-    "filterList": "Containing",
+    "any": "입력",
+    "closeTagTray": "태그 창 닫기",
+    "delete": "삭제 모드",
+    "filterList": "필터",
     "filterTags": "필터 태그",
-    "hidingFrequency": "빈도 숨기기",
-    "minimum": "minimum",
+    "hidingFrequency": "태그 개수 숨기기",
+    "minimum": "이상 포함된 태그",
     "presentTags": "이 Hub의 태그",
-    "showingFrequency": "빈도 보이기",
-    "showingMostSimilar": "비슷한 영상:",
+    "showingFrequency": "태그 개수 보이기",
+    "showingMostSimilar": "비슷한 동영상:",
     "sortAlphabetical": "이름순 정렬",
-    "sortByFrequency": "빈도로 정렬",
-    "starRating": "별점 등급",
+    "sortByFrequency": "태그 개수로 정렬",
+    "starRating": "별점 매기기",
     "timesPlayed": "재생 횟수"
   },
   "WIZARD": {
     "change": "변경",
     "clearRecent": "최근의 모든 기록 지우기",
-    "clipSnippets": "조각으로",
+    "clipSnippets": "개의 자투리 생성",
     "closeButton": "마법사 닫기",
     "createVideoHub": "동영상 Hub 만들기",
-    "demoVersion": "데모 버전은 50개 동영상으로 제한",
-    "eachSnippet": "각각의 조각들을",
-    "expected": "Expected:",
-    "extractClips": "클립 미리 보기 생성",
-    "found": "found",
-    "HubDirectory": "(Hub 디렉터리)",
-    "imported": "imported",
-    "importingFrom": "다음 위치에서 가져오기:",
+    "demoVersion": "50개 동영상으로 제한된 데모 버전",
+    "eachSnippet": "각각의 자투리를",
+    "expected": "예상:",
+    "extractClips": "",
+    "found": "찾음",
+    "hubDirectory": "(Hub 디렉터리)",
+    "imported": "삽입되는",
+    "importingFrom": "선택한 위치:",
     "importNew": "새 Hub 만들기",
     "loadFromFile": "Hub 파일 불러오기",
-    "minutes": "분마다",
-    "nameYourHub": "Hub 이름 지정",
-    "oneScreenshotEvery": "스크린샷 1개씩",
+    "minutes": "분마다 1회 스크린샷",
+    "nameYourHub": "Hub 이름",
+    "oneScreenshotEvery": "",
```

Comment:
367 + 369 -> minutes 1 screenshot every
369 Is there no problem even if it's blank?

Reply:
It's broken up because the sentence in English had to be:
![2020-02-22 11_11_57-Window](https://user-images.githubusercontent.com/17264277/75095577-3efbc780-5564-11ea-9976-0cb4be13c915.png)
If you leave it blank, there will be no word/phrase shown.
Current app in Korean
![image](https://user-images.githubusercontent.com/17264277/75095587-75394700-5564-11ea-8457-472bad4ccd5f.png)

I'll trust whatever you recommend 👍

Reply:
It's more natural to have nothing in five minutes.
![want](https://user-images.githubusercontent.com/57275671/75101069-64e3a380-561a-11ea-82c9-766e96968928.jpg)

I asked about the blanks (line 238, 360, 369) to make sentences more natural.

Reply:
Excellent! This works very well!

In English this would probably be "5 minutes take 1 screenshot" which isn't as elegant as "1 screenshot every 5 minutes" -- so I had to break up that phrase into 2 pieces.

If what you have is easy to understand -- we'll do as you recommend 👍 

Though when I translate `분마다 스크린샷` into English it is shown as [`Screenshot every minute`](https://translate.google.com/#view=home&op=translate&sl=auto&tl=en&text=%EB%B6%84%EB%A7%88%EB%8B%A4%20%EC%8A%A4%ED%81%AC%EB%A6%B0%EC%83%B7) ... 

I'm unsure if the translation is correct -- the selection isn't about how many screenshots to take every minute, but how many minutes pass before we take another screenshot.

Putting `5` in this selection box means _after 5 minutes another screenshot will be taken_, not that 5 screenshots will be taken every 1 minute.

So, with a `5`, for a 15 minute video, 3 screenshots will be taken, not 75.

Sorry about any confusion 😅

Reply:
I understood. I didn't think about it.
Thanks for the exponal. I'll fix it.
I'll also fix and update the blanks (line 238, 360, 369).